RadarSDK/Radar.m

```diff
@@ -489,10 +503,32 @@ + (void)logConversionWithName:(NSString *)name
 
 
 + (void)logConversionWithNotification:(UNNotificationRequest *)request {
+    [self logConversionWithNotification:request eventName: @"opened_app" conversionSource:@"notification" deliveredAfter: nil];
```

Comment:
Looks like we're always having `deliveredAfter` nil. I think we need to remove this param and use the registered at below in lieu of that, right?

Reply:
The `deliveredAfter` is currently unused as this PR excludes the on premise notification delivered piece. We still need that field.
We do not need a `registeredAt` field as that information is encoded into our identifier and can be inferred.

Reply:
Oh yes, duh. Makes sense.